src/main.rs

```diff
@@ -47,13 +47,18 @@ async fn main() -> Result<(), Box<dyn Error>> {
         Arc::new(sentry::transports::CurlHttpTransport::new(&options))
             as Arc<dyn sentry::internals::Transport>
     };
-    */
-    let _sentry = sentry::init(sentry::ClientOptions {
+     */
+    // Disable debug-images: it conflicts w/ our debug = 1 rustc build option:
+    // https://github.com/getsentry/sentry-rust/issues/574
+    let mut opts = sentry::apply_defaults(sentry::ClientOptions {
```

Comment:
bleh.

I get it, and 👍🏻,  but bleh.

(this seems like it really should be something configurable in the collector.)

Reply:
indeed 😃